.travis.yml

```diff
@@ -16,5 +16,5 @@ env:
   - DOCS_DIR: "../yeoman-generator-doc"
   - GH_OWNER: yeoman
   - GH_PROJECT_NAME: generator
-  - PACKAGE_NAME: yeoman-generator
+  - DEPLOY_ON_NODE_VERSION: v6
```

Comment:
You can define the Node version on which the docs should be generated and deployed in the `.travis.yml`.